run-go-fmt.sh

```diff
@@ -10,5 +10,6 @@ if ! command -v gofmt &> /dev/null ; then
 fi
 
 exec 5>&1
```

Comment:
Looks like we're not using /dev/fd/5 anymore, so this line can be removed from both scripts.

Reply:
done